docs/src/browsers.md

```diff
@@ -99,3 +99,17 @@ Google Chrome and Microsoft Edge respect enterprise policies, which include limi
 network proxy, mandatory extensions that stand in the way of testing. So if you are a part of the
 organization that uses such policies, it is the easiest to use bundled Chromium for your local testing,
 you can still opt into stable channels on the bots that are typically free of such restrictions.
+
+### How to install Google Chrome & Microsoft Edge?
```

Comment:
This actually hurts more than it helps, let's rather start with saying that GitHub Actions and other CIs have browsers pre-installed.